src/applications/vaos/services/var/index.js

```diff
@@ -236,7 +236,7 @@ export function submitAppointment(appointment) {
     method: 'POST',
     headers: { 'Content-Type': 'application/json' },
     body: JSON.stringify(appointment),
-  });
+  }).then(parseApiObject);
```

Comment:
Returns the attributes object that contains the data.

Reply:
I think this is where it is failing. While I do get a response preview with data in the local environment, when testing  the review instance in staging environment, the response code comes back as 204 No Content
```
Request URL: http://dcb43e8feb891345dae3bc67b4c602c0-api.review.vetsgov-internal/vaos/v0/appointments
Request Method: POST
Status Code: 204 No Content
Remote Address: 127.0.0.1:2001
Referrer Policy: no-referrer
Access-Control-Allow-Credentials: true
Access-Control-Allow-Methods: GET, HEAD, POST, PUT, PATCH, DELETE, OPTIONS
Access-Control-Allow-Origin: http://dcb43e8feb891345dae3bc67b4c602c0.review.vetsgov-internal
Access-Control-Expose-Headers: X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-Session-Expiration, X-CSRF-Token
Access-Control-Max-Age: 7200
Cache-Control: no-cache
Connection: keep-alive
Date: Wed, 05 Jan 2022 19:42:19 GMT
Referrer-Policy: strict-origin-when-cross-origin
Server: openresty
Set-Cookie: api_session=PtAxjV%2F3ZTvfUtqd52aZub9sJlzj3vqg7Y1dbjicAJuILlzYx7SyA3JkDaEOT7Ev5gQnbWrHDfpR0GueNpIv8F1mYhYvNdHRqa4ewzqa1qrgkEeweBn5c8THRRyWW29Tlx87VyApqXKfkJSrxva7tm8Z9H6dQbJON79z8S8IWUCR%2Fa01KVjbhDmuIf9HkpM0%2BqJ%2BLLwKA0cGNU%2BhFzCFwvXwN5YwmjScWMjcfb6xDgKwlEb1e%2BKV5IEcyXDTC5Rvn2K9V1wEYQioo9jiD4XkXaQQvLt9Jvpd1uYH3VxMnkje2B4isvtIjV7z7ot4FXHhMtn7sWzhMNAsx%2BRJF3E4TH9PlJqhwaz%2FFAoXixCx7wJS3mY%2ByWoydBv%2FD6Lq5qpzG52Q8Biekm%2FifUsqgjXlnGE1xnrML8ttzQUu6hkwJxcPTKDxjFqm%2Fn2FNfu6nZP4jUqx2sEHMNALkU%2BtBYx9%2FDBltalY4V3qRNdndJA5uc74Ke6ML9zjsbg5vYCBTVx4wxSxJ8yuRLKw8rwYTuEUQr%2FvIEtv%2BBsGebgF%2FLWnAM1caaxMIiclA25LDj%2Fu45VZ1OFV2tyF1iJVsoV1ZjEaDxdT4U8sOK82hGKPcjZ%2BTn25RWC6llO5QOkRRvovufGwSOUA19ZI9Qz0U9EYusxgqKXFzuItHsSGqzHpIFNR8Eqgcyr2%2B5A%2BmHv5Fg%2FXgbV1ihNQVbzN2kr63lvTvUeYprBLXLHzY0coEsxB5wGz2yVq%2Bw8tw0GS1nxu5CaSbnTMCBb8OcIVapikUjelNxQc1V5J5hptal4pTZNc--817oNcwO8FPfCpqh--fIQmTv%2BWJARybOZfgdjFuQ%3D%3D; path=/; HttpOnly; SameSite=Lax
Vary: Origin
X-Content-Type-Options: nosniff
X-CSRF-Token: f6IxNJ1IShotLLpgGz0pFyCkgyP/GM2YdjpVY6Orq2TaJ4Wb1mwXlSJuZjHpQ7X/5Z6AtaIjItllHJnjmdun9Q==
X-Download-Options: noopen
X-Frame-Options: SAMEORIGIN
X-Git-SHA: MISSING_GIT_REVISION
X-GitHub-Repository: https://github.com/department-of-veterans-affairs/vets-api
X-Permitted-Cross-Domain-Policies: none
X-Request-Id: 0f1dba36-91e8-44d8-ad64-bfd67509c748
X-Runtime: 1.058933
X-Session-Expiration: Wed, 05 Jan 2022 20:12:18 GMT
X-XSS-Protection: 1; mode=block
```